challenges/objects-arrays.js

```diff
@@ -55,11 +82,23 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i=0 ; i < graduates.length ; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (i=0 ; i < graduates.length ; i++) {
```

Comment:
This solution is extremely thorough, and it's cool to see you construct something more complex, but I'd like you to think about how to simplify this.